Makefile

```diff
@@ -172,9 +172,12 @@ CONFIG_DATA_FILE=$(DATA_DIR)/config.json
 INPUT_DIR=./input
 INPUT_FILE=$(INPUT_DIR)/input.json
 
+RELEASE_NAMESPACE=release.main
+
 POLICIES_DIR=./policies
 OPA_FORMAT=pretty
-OPA_QUERY=data.main.deny
+OPA_QUERY=data.$(RELEASE_NAMESPACE).deny
```

Comment:
With conftest you can just specify the namespace. This might only select the deny rules, I'm not sure though.